docs/versioned_docs/version-v1.3.0/02-Features/02-message-handling/04-customize-general.md

```diff
@@ -0,0 +1,259 @@
+---
+title: "Customize general message handling"
+layout: default
+---
+
+# Customize general message handling
+
+While the message processing is handled by the `IMessageHandler<>` implementations, the message router controls in what format the message is received.
+We allow several customizations while using a built-in or implementing your own message router.
+
+## Filter messages based on message context
```

Comment:
This information is already available in the 01-, 02- and 03 documents.  Is it necessary to mention it here as well ?

Reply:
Not sure what you mean here? The way the feature documentation is set up is that each version builds upon the next. So, it is not so that the latest version only contains the documentation of the new features present in that version.

Reply:
Answered this somewhere else: it wasn't clear before that this was possible, while now, we provide this on Azure resource type. It is not duplicated in the Azure Functions variant, as those are the same as the regular worker.